### PR TITLE
feat: 0.4.0 setup with askar and indy-sdk

### PR DESCRIPTION
**Pull request by TimoGlastra**
Adds Askar and Indy setup for 0.4.0. Doesn't yet update the tutorials and update instructions for Indy VDR / AnonCreds RS.

See video for overview of updates: https://www.loom.com/share/59fdc8cac23b439495f5796347ff2097

### CODE REVIEW

File: guides/getting-started/set-up/indy-sdk/linux.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Linux
+
+To install [Indy SDK](https://github.com/hyperledger/indy-sdk) on Linux a couple of dependencies are required. This guide covers the installation for the more popular Linux distributions.
```
Comment (Vickysomtee) [inline guides/getting-started/set-up/indy-sdk/linux.md:3]:
```suggestion
To install [Indy SDK](https://github.com/hyperledger/indy-sdk) on Linux, a couple of dependencies are required. This guide covers the installation for the more popular Linux distributions.
```

File: guides/getting-started/set-up/indy-sdk/macos-arm.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# macOS (ARM)
+
+To install [Indy SDK](https://github.com/hyperledger/indy-sdk) on macOS a couple of dependencies are required. This guide covers the installation of the Indy SDK for macs with an ARM processor. For installing the Indy SDK on Intel based macs, please refer to the [Indy SDK macOS (Intel)](./macos-intel) guide.
```
Comment (Vickysomtee) [inline guides/getting-started/set-up/indy-sdk/macos-arm.md:3]:
```suggestion
To install [Indy SDK](https://github.com/hyperledger/indy-sdk) on macOS, a couple of dependencies are required. This guide covers the installation of the Indy SDK for macs with an ARM processor. For installing the Indy SDK on Intel based macs, please refer to the [Indy SDK macOS (Intel)](./macos-intel) guide.
```

File: guides/tutorials/create-a-connection.md [hunk 3]
```diff
@@ -15,6 +8,15 @@ this invitation and at that point they have established a connection and they
 know how to reach each other for sending a basic message, issuing a credential,
 verifying a proof, etc.
 
+:::info
+
+This section assumes that
+
+1. You have [set-up your develoment environment](../getting-started).
+1. You have basic knowledge of the required fields in the [Agent Config](./agent-config)
```
Comment (Vickysomtee) [inline guides/tutorials/create-a-connection.md:16]:
```suggestion
1. You have [set-up your develoment environment](../getting-started).
2. You have basic knowledge of the required fields in the [Agent Config](./agent-config)
```

File: guides/tutorials/postgres-database-nodejs/windows.md [hunk 4]
```diff
@@ -5,7 +5,7 @@
 1. Download and install Visual Studio Community Edition 2022
 1. Install git for windows
 1. Download rust for windows [here](https://www.rust-lang.org/en-US/install.html)
```
Comment (Vickysomtee) [inline guides/tutorials/postgres-database-nodejs/windows.md:7]: This is supposed to be numbered list, right? If yes, we will have to update the list numbers (2, 3, 4)

File: guides/tutorials/postgres-database-nodejs/windows.md [hunk 5]
```diff
@@ -5,7 +5,7 @@
...
-1. Make sure you have already setup a libindy for windows from [here](../../getting-started/installation/nodejs/windows.md)
+1. Make sure you have already setup alibindy for windows from [here](../../getting-started/indy-sdk/windows)
```
Comment (Vickysomtee) [inline guides/tutorials/postgres-database-nodejs/windows.md:8]:
```suggestion
1. Make sure you have already setup a libindy for windows from [here](../../getting-started/indy-sdk/windows)
```

**Review by Vickysomtee — COMMENTED** [review]
A little review on some spelling errors. Also, for formal documentation, I think it is better to minimise the use of apostrophe? i.e. "You are" instead of "You're"

File: guides/tutorials/postgres-database-nodejs/windows.md (hunk 4 above)
Comment (TimoGlastra) [reply guides/tutorials/postgres-database-nodejs/windows.md:7]: In markdown, numbered lists are added automatically if you give all items a 1. prefix. It's handy for when you change the order / add / remove items, as you don't have to update the numbers

**Review by blu3beri — APPROVED** [review]
I really like the new changes to the docs! Looks very nice with the cards and such!!

**Pull request merged**